server/test/routes/createApp.js

```diff
@@ -92,5 +87,34 @@ describe('v1/apps', () => {
             const res = await server.inject(request)
             expect(res.statusCode).to.equal(400)
         })
+
+        it('should create app when app-bundle is in top-level directory', async () => {
+            const form = new FormData()
+            form.append('app', JSON.stringify(sampleApp))
+            form.append(
+                'file',
+                fs.createReadStream(
+                    path.join(__dirname, '../', 'sample-app-top-level.zip')
```

Comment:
when I open up this zip file, it didn't look like there was an extra top-level layer? So I'm not sure how the test is different than the `should create an app` (but maybe I'm missing something or this is due to my OS or something)?